ApiBundle/Exceptions/HttpException/TransformParameterTypeHttpTypeHttpException.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\Exceptions;
+
+/**
+ * Class TransformerParameterException
+ */
+class TransformerParameterTypeHttpException extends \Exception
+{
+    const DEVELOPER_MESSAGE  = 'open_orchestra_api.transformer.type';
```

Comment:
As this exception should not be thrown on user (and so serialized), I do not think that you need all those constant